app/services/pending_action_service/reminders.rb

```diff
@@ -4,30 +4,38 @@ class PendingActionService
   class Reminders
     class << self
       def send
-        PendingAction
+        actions = PendingAction
           .limit(10)
           .not_confirmed
           .only_emailed_once
-          .not_emailed_last_24.each do |action|
-            assigns = {
-              token: action.token,
-              email: action.email,
-              name: action.data['name'],
-              page: Page.find(action.data['page_id'])
-            }
+          .not_emailed_last_24
 
-            EmailDelivery.send_email(action, html: html(assigns),
-                                             text: text(assigns),
-                                             subject: I18n.t('double_opt_in.email.subject', locale: :de))
-          end
+        actions.each do |action|
```

Comment:
Does this mean we resend an opt in email for every action they've taken in the last 24 hours if they haven't consented yet?